src/backtrace.rs

```diff
@@ -0,0 +1,95 @@
+// SPDX-FileCopyrightText: 2023 Rivos Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+use alloc::fmt::{Display, Formatter, Result};
+/// Get an array of backtrace addresses.
```

Comment:
nit: Not sure if the project enforces a particular style for use statements (grouping by external/internal) but I doesn't look right to have a comment in the middle here.

Reply:
fixed